pkg/engine/update.go

```diff
@@ -43,14 +45,44 @@ type RequiredPolicy interface {
 	Install(ctx context.Context) (string, error)
 }
 
+// LocalPolicyPack represents a set of local Policy Packs to apply during an update.
+type LocalPolicyPack struct {
+	// Name provides the user-specified name of the Policy Pack.
+	Name string
+	// Path of the local Policy Pack.
+	Path string
+}
+
+// ConvertPathsToLocalPolicyPacks is a helper function for converting the list of local Policy
+// Pack paths to list of LocalPolicyPack.
+func ConvertPathsToLocalPolicyPacks(localPaths []string) []LocalPolicyPack {
```

Comment:
The name here seems a little awkward. What do you think of `MakeLocalPolicyPacks(...)` instead? Also, we should call out in the function's doc comment that the `Name` field isn't set. (And will need to be initialized later.) As it's easy to assume that the returned `LocalPolicyPack` struct will be fully-initialized, which isn't the case.